contracts/solidity/deploy.sh

```diff
@@ -16,5 +16,4 @@ load_env() {
 }
 load_env
 
-echo -e "${GREEN}\n=> [ETH] Deploy Escrow${COLOR_RESET}"
```

Comment:
Why did you remove this? Isn't this better for clarity in the deploying process

Reply:
reverted :back: